Journalator_Display/Source/Display/InvestONator/PortfolioDisplay.lua

```
@@ -0,0 +1,426 @@
+      end
+      frame:SetParent(nil)
+      table.insert(self.PortfolioFramePool.free, frame)
+    end
+  end
+  self.PortfolioFramePool.inUse = {}
+end
+
+-- Creates and configures the portfolio creation dialog used to add new portfolios.
+-- The dialog includes inputs for portfolio name and total investment, plus Create and Cancel buttons.
+-- Stores dialog and input widgets on the mixin as `CreateDialog`, `NameEditBox`, and `InvestmentEditBox`.
+function JournalatorInvestONatorPortfolioDisplayMixin:SetupCreatePortfolioDialog()
```

Luacheck (1.2.0)

[warning] mutating non-standard global variable 'JournalatorInvestONatorPortfolioDisplayMixin'

(W112)